site/_shortcodes/AndroidBrowserSupportTable.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {html} = require('common-tags');
+
+/**
+ * Generates HTML table for Android Browser feature comparison
+ *
+ * @param {string} _method the method being referenced in the table
+ * @param {string} chrome configuration data for chrome
+ * @param {string} edge configuration data for microsoft edge
+ * @param {string} firefox configuration data for firefox
+ * @param {string} opera configuration data for opera
+ * @param {string} samsung configuration data for samsung internet
+ * @param {string} brave configuration data for brave
+ * @param {string} tor configuration data for tor browser
+ * @param {string} uc configuration data for uc
+ * @return {string}
+ */
+
+const BROWSERS = {
+  chrome: {
+    name: 'Chrome',
+    link: 'https://play.google.com/store/apps/details?id=com.android.chrome',
+  },
+  edge: {
+    name: 'Microsoft Edge',
+    link: 'https://play.google.com/store/apps/details?id=com.microsoft.emmx',
+  },
+  firefox: {
+    name: 'Firefox',
+    link: 'https://play.google.com/store/apps/details?id=org.mozilla.firefox',
+  },
+  opera: {
+    name: 'Opera',
+    link: 'https://play.google.com/store/apps/details?id=com.opera.browser',
+  },
+  samsung: {
+    name: 'Samsung Internet',
+    link: 'https://play.google.com/store/apps/details?id=com.sec.android.app.sbrowser',
+  },
+  brave: {
+    name: 'Brave',
+    link: 'https://play.google.com/store/apps/details?id=com.brave.browser',
+  },
+  tor: {
+    name: 'Tor',
+    link: 'https://play.google.com/store/apps/details?id=org.torproject.torbrowser',
+  },
+  uc: {
+    name: 'UC',
+    link: 'https://play.google.com/store/apps/details?id=com.UCMobile.intl',
+  },
+};
+
+const COMPAT = {
+  y: {
+    short: 'yes',
+    label: 'Supported',
```

Comment:
This, together with the other plain strings in here, would ideally go through the `i18n` filter. See the [BrowserCompat implementation](https://github.com/GoogleChrome/webdev-infra/blob/cd4a224d386e8d0f270e651a8deefafe2259ad8f/shortcodes/BrowserCompat/index.js#L57) for an example.

You'll get the locale via `this.ctx.locale` in `AndroidBrowserSupportTable` if you make it a proper function instead of an arrow function.

Though given this page will probably be never translated for multiple reasons I would also be fine leaving the strings here as is. Up to you!